src/victorops.coffee

```diff
@@ -187,6 +187,8 @@ class VictorOps extends Adapter
       for item in data.PAYLOAD.TIMELINE_LIST
         try
           if item.ALERT
+            @robot.brain.set item.ALERT["INCIDENT_NAME"], item.ALERT
```

Comment:
I have situations where I want my hubot to take action on incidents even after they are resolved. If there was a way to just broadcast all alerts and let my script act on that alert, I could work with that.